frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/config.ts

```diff
@@ -8,6 +8,6 @@ export function getDefaultDisplayOptions(
     theme: "light",
     bordered: true,
     titled: true,
-    hide_download_button: shouldShownDownloadData ? false : null,
+    downloads: shouldShownDownloadData ? true : null,
```

Comment:
Good point, fixed

Reply:
I take it back, it must be null to be omitted in the url in non-paid plans, luckily a [unit test failed](https://github.com/metabase/metabase/actions/runs/9942676495) and I realized it 😅

Reply:
there's another way to look at this. Either you keep the data as we want to represent (`null` for omitting it in the URL) or we have a layer that translates the underlying data to the desired shape (e.g. omitting it if the values are default, or is non-paid plan). I find mixing the data and presentation logic could make it harder to reason about.

I found that having a layer that translates the data to the desired output makes storing data a lot cleaner and makes it easier to reason about.

For example, we hide some embed options when they're default values. We don't store them as `null` but we remove them before creating the URL.

https://github.com/metabase/metabase/blob/d37aaedb3e10ae703a03d8f210ae19fb63a67033/frontend/src/metabase/public/lib/code-templates.ts#L11-L17

Reply:
Oh I see, looking at the tests for the analytics events it seems that we expect null to be sent though, and I'm afraid changing this would break it. To stay safe I'd just the logic as it is as I just edited what was already there, I'll double check if we can do something about it after I write the new analytics code+tests so it's easier to see if I mess something up